### ISSUE TITLE
cts files emitting cjs files with ES module syntax

### ISSUE THREAD

**Issue by PythonCoderAS** [microsoft/TypeScript#56999]
### 🔎 Search Terms

cts
cjs
.cts
.cjs

### 🕗 Version & Regression Information

- This is the behavior in every version I tried, and I reviewed the FAQ for entries about cts files

### ⏯ Playground Link

https://github.com/PythonCoderAS/typescript-bug-repro

### 💻 Code

File `config.ts`:

```ts
import { resolve } from "path";

const config = resolve(__dirname, "config.cts")

module.exports = config;
```

### 🙁 Actual behavior

`config.cjs`:

```js
import { resolve } from "path";
const config = resolve(__dirname, "config.cts");
module.exports = config;
//# sourceMappingURL=config.cjs.map
```

### 🙂 Expected behavior

```js
const { resolve } = require("path");
const config = resolve(__dirname, "config.cts");
module.exports = config;
//# sourceMappingURL=config.cjs.map
```

This is correct because cjs files cannot use ESM things like import.

### Additional information about the issue

This might be an error to use but it should throw an error instead of failing silently.

**Comment by RyanCavanaugh** [comment]
Duplicate #50647
